utilities/copyoffload_migration.py

```
@@ -533,32 +599,58 @@ def verify_xcopy_used(
 
-    for pod in populate_pods:
-        pvc_name: str = pod.instance.metadata.labels.get("pvcName", pod.name)
-        log_content: str = pod.log()
+    # Try to use cached logs first (for MTV builds that cleanup pods quickly)
+    cached_logs: list[dict[str, Any]] | None = fixture_store.get("populate_pod_logs", {}).get(migration_uid)
+
+    # Try to use cached logs first (for MTV builds that cleanup pods quickly)
+    pod_logs: list[dict[str, str]] = []
+
+    if cached_logs:
+        LOGGER.info(f"Using {len(cached_logs)} cached populate pod log(s)")
```

Ruff (0.15.15)

[warning] Logging statement uses f-string

(G004)